src/Microsoft.DotNet.CodeFormatter.Analyzers.Tests/AnalyzerFixerTestBase.cs

```diff
@@ -33,23 +33,21 @@ private IFormattingEngine Engine
             }
         }
 
-        protected void DisableAllDiagnostics()
+        protected PropertyBag BuildAllRulesDisabledPolicy()
```

Comment:
ConstructPolicyThatDisablesAllAnalyzers

Perfectly descriptive and irritatingly prolix. ship it?